src/main/java/greedy/greedybot/presentation/jda/listener/ScheduledMessageScheduler.java

```diff
@@ -0,0 +1,50 @@
+package greedy.greedybot.presentation.jda.listener;
+
+import greedy.greedybot.application.message.dto.ScheduledMessage;
+import greedy.greedybot.domain.message.ScheduledMessageRepository;
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.channel.concrete.TextChannel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+
+@Component
+public class ScheduledMessageScheduler {
+
+    private final Logger log = LoggerFactory.getLogger(StatusCommandListener.class);
```

Comment:
https://github.com/greedy-team/greedy-discord-bot/pull/19#discussion_r1978680846
```suggestion
    private final Logger log = LoggerFactory.getLogger(ScheduledMessageScheduler.class);
```